pkg/operator/starter.go

```diff
@@ -142,11 +142,11 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 			"v4.1.0/kube-apiserver/localhost-recovery-client-crb.yaml",
 			"v4.1.0/kube-apiserver/localhost-recovery-sa.yaml",
 			"v4.1.0/kube-apiserver/localhost-recovery-token.yaml",
-			"v4.1.0/kube-apiserver/audit-policies-cm.yaml",
 			"v4.1.0/kube-apiserver/apiserver.openshift.io_apirequestcount.yaml",
 			"v4.1.0/alerts/api-usage.yaml",
 			"v4.1.0/alerts/cpu-utilization.yaml",
 			"v4.1.0/alerts/kube-apiserver-requests.yaml",
+			"libgoaudit.AuditPoliciesConfigMapFileName",
```

Comment:
not a literal, it should not be quoted, plain `libgoaudit.AuditPoliciesConfigMapFileName`